test/fixtures/autofix/jQuerySap.js

```diff
@@ -0,0 +1,237 @@
+	jQuery.sap.measure.clear();
+	var filteredEntries = jQuery.sap.measure.filterMeasurements((o) => { return o.categories.indexOf("categoryX") > -1 ? o : null; });
+	var allEntries = jQuery.sap.measure.getAllMeasurements(true);
+	var entry = jQuery.sap.measure.getMeasurement("measureB");
+	var pausedEntry = jQuery.sap.measure.pause("measureB");
+	var resumedEntry = jQuery.sap.measure.resume("measureB");
+	var isActive = jQuery.sap.measure.getActive();
+	var isStillActive = jQuery.sap.measure.setActive(true, ["categoryX", "categoryZ"]);
+	jQuery.sap.measure.remove("measureA");
+	var myFunctionObject = { myFunction: () => console.log("test") };
+	var isRegisterSuccessful = jQuery.sap.measure.registerMethod("measureC", myFunctionObject, "myFunction");
+	var isUnregisterSuccessful = jQuery.sap.measure.unregisterMethod("measureC", myFunctionObject, "myFunction");
+	jQuery.sap.measure.unregisterAllMethods();
+	var timings = jQuery.sap.measure.getRequestTimings();
+	jQuery.sap.measure.clearRequestTimings();
+	jQuery.sap.measure.setRequestBufferSize(100)
+
+	// https://github.com/SAP/ui5-linter/issues/561
+	await jQuery.sap.fesr.setActive(true);
+	var isFESRActive = jQuery.sap.fesr.getActive();
+
+	jQuery.sap.fesr.addBusyDuration(33);
+	await jQuery.sap.interaction.setActive(true);
+	var isInteractionTrackingActive = jQuery.sap.interaction.getActive();
+	jQuery.sap.interaction.notifyStepStart("startup", "startup", true);
+	jQuery.sap.interaction.notifyStepEnd(true);
+	jQuery.sap.interaction.notifyEventStart(myClickEvent);
+	jQuery.sap.interaction.notifyScrollEvent(myScrollEvent)
+	jQuery.sap.interaction.notifyEventEnd();
+	jQuery.sap.interaction.setStepComponent("component-fancy");
+	jQuery.sap.measure.startInteraction("click", myButton);
+	jQuery.sap.measure.endInteraction(true);
+	jQuery.sap.measure.clearInteractionMeasurements();
+	var filteredMeasurements = jQuery.sap.measure.filterInteractionMeasurements(() => true);
+	var allMeasurements = jQuery.sap.measure.getAllInteractionMeasurements();
+	var pendigngMeasurement = jQuery.sap.measure.getPendingInteractionMeasurement();
+
+	var transactionId = jQuery.sap.fesr.getCurrentTransactionId();
+	var rootId = jQuery.sap.fesr.getRootId()
+	jQuery.sap.passport.setActive(true);
+	var traceLevel = jQuery.sap.passport.traceFlags("medium");
+
+	// https://github.com/SAP/ui5-linter/issues/562
+	var frameOptions = new jQuery.sap.FrameOptions();
+
+	// https://github.com/SAP/ui5-linter/issues/563
+	// Do not migrate this case.
+	// sap/ui/util/ActivityDetection is a private module!
+	var isActivityDetectionActive = jQuery.sap.act.isActive();
+
+	jQuery.sap.initMobile();
+	jQuery.sap.setIcons({
+		'phone': 'phone-icon_60x60.png',
+		'phone@2': 'phone-retina_120x120.png',
+		'tablet': 'tablet-icon_76x76.png',
+		'tablet@2': 'tablet-retina_152x152.png',
+		'precomposed': true,
+		'favicon': 'desktop.ico'
+	});
+	jQuery.sap.setMobileWebAppCapable(true);
+
+	// TODO: Requires couple runs!
```

Comment:
Is this TODO still valid?

Also, the replacement of the static functions is not covered here. It works, but is not replaced as I would expect it. See openui5 QUnit test "Storage API using static methods".

Reply:
What's the status here? Looking at the current state, I don't fully understand why the one line can be fully replaced by the other can't. Anyways, this comment needs to be updated.

Reply:
The one is replaced with `exportNameToBeUsed` and the other one with `exportCodeToBeUsed`.

The issue with `exportCodeToBeUsed` is that currently we replace the whole expression, including braces and wipe out everything inside as arguments might have different positions, be gone or something else.

I think we could solve these cases along with `dellayedCall` migration that we postponed.